js/core.js

```diff
@@ -1055,6 +1055,20 @@
          */
         this._showSavedQuery = _getHierarchicalDisplayAnnotationValue(this, "show_saved_query");
 
+        /**
+         * @desc The path to the table where the favorite terms are stored
+         * @type {string}
+         */
+        this._favoritesPath = null;
+        if (this.annotations.contains(module._annotations.TABLE_CONFIG)) {
+            var userFavorites = this.annotations.get(module._annotations.TABLE_CONFIG).content.user_favorites;
+            console.log(userFavorites);
+            if (userFavorites && userFavorites.storage_table) {
```

Comment:
You should make sure `userFavorites.storage_table` is an Object, and all the `.catalog`, `.schema`, `.table` are non empty strings.

Reply:
Also don't forget to remove the console.log

Reply:
For the favoritesPath, in the case of create, I’m using the the path as is to create a reference. But for delete I’m appending 2 filters to the path (`user_id` and `table.name`) before creating the reference.